code/modules/events/anomaly.dm

```diff
@@ -24,8 +24,8 @@
 		/area/holodeck,
 		/area/shuttle,
 		/area/maintenance,
-		/area/science/test_area)
-		)
+		/area/science/test_area,
```

Comment:
What is with these new commas at the end of some lists?

Reply:
was discussed in discord but if anyone in tha future is reading the PR:
This makes it so new additions to the lists count as only 1 addition instead of 2 additions and 1 deletion